torchci/lib/drciUtils.ts

```diff
@@ -476,3 +465,42 @@ export function hasSimilarFailuresInSamePR(
 
   return;
 }
+
+export async function getPRMergeCommits(
+  owner: string,
+  repo: string,
+  prNumbers: number[]
+): Promise<Map<number, string[]>> {
+  // Sort by comment ID desc because we don't want to depend on _event_time in
+  // general
+  const query = `
+SELECT
+  pr_num,
+  merge_commit_sha,
+FROM
+  default.merges
```

Comment:
Is `final` needed here?

Reply:
done, idk who's going to be modifying this table after insert though